### DISCUSSION TITLE
Startup command for Fluent-Bit v3.1.2 on Windows

### DISCUSSION THREAD

**Discussion by pkqsun** [fluent/fluent-bit#9621] [Q&A]
Hello Everyone,

The background is that we use chocolatey tools to re-pack fluent-bit MSI binary and generate one nupkg file with customized configuration in it. 
After a buck provisioning, all windows servers (500+) will be installed fluent-bit, then all of them will start to collect local windows event/software logs and sent logs to one bastion host without any interventions.

From Oct, we used v3.1.9 with classic mode conf files and it almost finished our requirements. However, as v3.2.1 identified, classic mode will be deprecated at end of 2025, so we change to use v3.2.1 from now.

BTW, for v3.2.1, the default installed configuration file still use .conf of classic mode. We deleted all of them and replaced with new .yaml ones.

In short, my question is that with fluent-bit.yaml, how to change the startup service command for MSI installed case 
**from "$PATH\fluent-bit.exe -c $PATH\fluent-bit.conf" 
to "$PATH\fluent-bit.exe -c $PATH\fluent-bit.yaml"** 
after fluent-bit MSI installed.

I ever find INSTALL_ROOT parameter from MSI log file to re-located the installation path, but for above change, still have no idea of which parameter would work for me. 

Appreciate for any help on this.

**Comment by pkqsun** [comment]
I know I could remove the fluent-bit service and then re-define it with new startup cmd, but we consider it as plan B and prefer to modify the service before or during installation if anyone has related experience.

**Reply by patrick-stephens** [reply]
You're right, we should transition the config file to YAML by default @cosmo0920 .
Cmake handles the installer generation btw so feel free to submit a PR: https://github.com/fluent/fluent-bit/blob/8994bfcde5175e7aa6fa2088b1f2be44de1b240b/CMakeLists.txt#L1324-L1340
